modules/identity-provider-config-map.adoc

```diff
@@ -35,6 +35,23 @@ authority must be stored in the `ca.crt` key of the `ConfigMap` object.
 ----
 $ oc create configmap ca-config-map --from-file=ca.crt=/path/to/ca -n openshift-config
 ----
++
+[TIP]
+====
+You can alternatively apply the following YAML to create the config map:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ca-config-map
+  namespace: openshift-config
+type: Opaque
+data:
+  ca.crt: <base64_encoded_CA_certificate_PEM>
```

Comment:
```
ca.crt: |
    <value>
```
the pipe is a notation for a multiple-lines input. But yes, that's probably the better option here since the certificate's PEM is going to span across multiple lines :+1:

Reply:
Thanks, updating here and in the other places in this file that use the ca.crt field.

Reply:
Also, IIRC the PEM is not base64-encoded here, that might have been a mistake from my side.